src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,34 @@
+import {Component, Input} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{striped && 'progress-striped'}} {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePercentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  private _striped: boolean;
```

Comment:
What about animated stripes (http://v4-alpha.getbootstrap.com/components/progress/#animated-stripes). This seems to be a pure CSS thing in BS4